Source/Core/VideoCommon/VideoConfig.h

```diff
@@ -58,173 +58,173 @@ struct VideoConfig final
   void VerifyValidity();
 
   // General
-  bool bVSync;
-  bool bVSyncActive;
-  bool bWidescreenHack;
-  AspectMode aspect_mode;
-  AspectMode suggested_aspect_mode;
-  bool bCrop;  // Aspect ratio controls.
-  bool bShaderCache;
+  bool bVSync{};
```

Comment:
Throughout most of the files you used `bool x = false` or `int x = 0`.  Here you use `bool x{}` and `int x{}`.  Can we make this consistent?

Reply:
I've changed them to `= false` or `= 0`.